org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/AcquisitionDevice.java

```diff
@@ -296,6 +290,10 @@ public void run(IPosition parent) throws ScanningException, InterruptedException
 			RunnableDeviceServiceImpl.setCurrentScanningDevice(null);
 		}
 	}
+	
+	private void positionComplete(IPosition pos) throws EventException, ScanningException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)